andela/wg-ravens
# Ft deactivated users page 157589639

#### What does this PR do?
Divide view of gym members into active and inactive users

#### Description of Task to be completed?
Gym members have only been visible all together in one view and the only way to find out the activity status of a member has been by opening their descriptive overview. Only active users should be visible in the default view with an option of viewing the inactive members on a different Gym member list of inactive members

#### How should this be manually tested?
Login to wger as a gym manager and select a gym to list members. You should only be able to see active members and a link to inactive members.

#### What are the relevant pivotal tracker stories?
#157589639
